modules/sbt-coursier/src/main/scala/coursier/sbtcoursier/UpdateTasks.scala

```diff
@@ -133,7 +133,8 @@ object UpdateTasks {
               includeSignatures,
               sbtBootJarOverrides,
               classpathOrder = true,
-              missingOk = sbtClassifiers
+              missingOk = sbtClassifiers,
+              classLoaders = ??? // todo
```

Comment:
I don't understand when this is used

Reply:
Just pass `Nil`, this corresponds to the `sbt-coursier` plugin, which is basically deprecated now...